cfme/tests/configure/test_default_views_infra.py

```diff
@@ -34,9 +33,8 @@ def setup_a_provider():
     'VMs': Vm,
     'My Services': MyService,
     'Catalog Items': CatalogItem,
-    'VMs & Instances': 'service_vms_instances',
-    'Templates & Images': 'service_templates_images'
-}
+    'VMs & Instances': VmsInstances,
+    'Templates & Images': TemplatesImages}
```

Comment:
@izapolsk that was done to fit others locations in that test. it has further some login with conditions which changes destination.